### ISSUE TITLE
Find A11Y issues on the docs site

### ISSUE THREAD

**Issue by Idrinth** [Idrinth/api-bench#923]
## Is your research related to a problem? Please describe

Our accessibility may not be perfect.

## Describe question you have

Find issues we have on the docs site based on the accessibility checklist of A11Y. Please raise an issue for anything you find.

## Additional context

https://www.a11yproject.com/checklist/

**Comment by marikadeveloper** [comment]
There's also this tool https://github.com/dequelabs/axe-core that can be used in unit tests like this:

```
it('should be accessible', async () => {
  const { container } = render(<Component />);
  const results = await axe(container);

  expect(results).toHaveNoViolations();
});
```

**Comment by Idrinth** [comment]
> There's also this tool https://github.com/dequelabs/axe-core that can be used in unit tests like this:
> 
> ```
> it('should be accessible', async () => {
>   const { container } = render(<Component />);
>   const results = await axe(container);
> 
>   expect(results).toHaveNoViolations();
> });
> ```

That sounds perfect!

**Comment by itsmacr8** [comment]
If I understand correctly, if we check the container component, it will evaluate all the elements inside it for issues since everything is inside it. Am I right? @marikadeveloper

**Comment by Idrinth** [comment]
> If I understand correctly, if we check the container component, it will evaluate all the elements inside it for issues since everything is inside it. Am I right? @marikadeveloper

I believe it would be best to add a test like this to each component to make pinpointing easier.

**Comment by itsmacr8** [comment]
I checked with the browser extension from that mentioned tool. I think, we have problems with the nav menu (contrast ratio). If I remember correctly, we will get help from the design team. So, we can leave it for later. Am I right?

**Comment by Idrinth** [comment]
> I checked with the browser extension from that mentioned tool. I think, we have problems with the nav menu (contrast ratio). If I remember correctly, we will get help from the design team. So, we can leave it for later. Am I right?

agreed, we don't need a solution yet, but we could already integrate the tests

**Comment by marikadeveloper** [comment]
@itsmacr8 do you have time to integrate the tests? If not, I can do it, let me know😊

**Comment by itsmacr8** [comment]
> @itsmacr8 do you have time to integrate the tests? If not, I can do it, let me know😊

Go ahead, @marikadeveloper. We have problems with the nav menu (contrast ratio). we will get help from the design team. So, make sure to add the **warning parameter or silence the errors for now,** otherwise it will throw errors.
